src/_guides/libraries/private-files.md

```diff
@@ -22,8 +23,9 @@ created by pub:
 
 {% prettify none tag=pre+code %}
 .dart_tool/
-.packages
+.packages (deprecated)
 build/
+package_config.json
```

Comment:
`package_config.json` is part of the `.dart_tool/` so I don't think we need this mention.

```suggestion
```